bool request_map
---
geometry_msgs/Point32[] map
